app/models/invoice.rb

```diff
@@ -45,7 +51,7 @@ def subtotal
     if line_items.any?
       line_items.offers.map(&:amount).sum
```

Comment:
If applied_coupon? is a ? method, shouldn't it only respond with bool? maybe we should need to rename this method, as i'm unclear why applied_coupon?  can respond with the total amount paid. seems like that should be a different method.

also i don't love the if/else/ternary operator
maybe we could make the first line a return, like: 
`return line_items.offers.map(&:amount).sum if line_items.any?`
and simplify the whole thing?

Reply:
Applied coupon returns a boolean!